installers/rpm-code/dev-env.sh

```diff
@@ -1,12 +1,27 @@
 #!/bin/bash
 # intended to be sourced
+cd "$(dirname "${BASH_SOURCE[0]}")"
+_root() {
+    local r=$PWD
+    while [[ $(basename $r) != src ]]; do
+        d=$(dirname "$r")
+        echo $d
```

Comment:
I just tested this and it seemed like all these extra echos are being added to the value of repo_fedora_dir

Reply:
5fedd36